virttest/env_process.py

```diff
@@ -62,6 +62,14 @@
 _setup_manager = test_setup.SetupManager()
 
 
+#: Hooks to use for own customization stages of the virtual machines
+#: with params and env as supplied arguments
```

Comment:
Should add the argument `test` as well, to make it consistent with the callers.

Reply:
Added, including a comment "or others of your choosing".